.github/workflows/test.yml

```diff
@@ -200,6 +239,8 @@ jobs:
           sudo apt-get install -y expect
       - name: "Ensure that the integration test suite is compatible with Docker"
         run: go test -timeout 20m -v -exec sudo ./cmd/nerdctl/... -args -test.target=docker -test.kill-daemon
+      - name: "Ensure that the IPv6 integration test suite is compatible with Docker"
+        run: go test -timeout 20m -v -exec sudo ./cmd/nerdctl/... -args -test.target=docker -test.kill-daemon -test.ipv6
```

Comment:
Doesn't this need reconfiguring sysctl and `/etc/docker/daemon.json` ?

Reply:
No. Fun fact 😂 
![image](https://github.com/containerd/nerdctl/assets/25029451/6a47474a-cab3-401e-be59-eedf1afadabc)

Reply:
Somehow the integration-tests-ipv6 fails to run nested inside docker and qemu, despite sysctl and daemon being configured. Later @Zheaoli  figured out a solution by using `network=host`  to run it successfully. However, this leads to non-IPv6 test failure. So I have to separate the IPv6 test.

Reply:
hmm, it should be able to run nested, but nested networks are complex and hard to debug, maybe you are missing certain sysctl or something, who knows